<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(180,180,180);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="editLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(90,90,90);</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(130,130,130)</string>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(130,130,130)</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80);
color: rgb(255, 246, 235);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="plateInput">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80);
color: rgb(255, 246, 235)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="renterNameInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80);
color: rgb(255, 246, 235);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoInput">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80);
color: rgb(255, 246, 235);</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(110,110,110);</string>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="plateLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(110,110,110);</string>
    </property>
    <property name="text">
     <string>Plate Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="phoLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(110,110,110);</string>
    </property>
    <property name="text">
     <string>Renter's Contact Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(110,110,110);</string>
    </property>
    <property name="text">
     <string>Renter's Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="promptLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(110,110,110);</string>
    </property>
    <property name="text">
     <string>Is it rented?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="statusButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(120,120,120);</string>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QLabel" name="startLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(110,110,110);</string>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="endLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(110,110,110);</string>
    </property>
    <property name="text">
     <string>End Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDateEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80);
color: rgb(255, 246, 235);</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDateEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80);
color: rgb(255, 246, 235)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit_Page">
    <property name="title">
     <string>Edit Page</string>
    </property>
   </widget>
   <addaction name="menuEdit_Page"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
